backend/services/goals_service.py

```
@@ -231,3 +238,116 @@ def check_and_populate_goals(self, user_id: int):
+            return f"Complete {requirement} modules"
+        if type == MetricType.EARN_GEMS:
+            return f"Earn {requirement} gems"
+        if type == MetricType.EXTEND_STREAK:
+            if requirement == 1:
+                return "Extend your streak by 1 day"
+            else:
+                return f"Practice on {requirement} days"
+        return None
+
+    @staticmethod
+    def _is_goal_req_in_range(type, time_period, requirement):
```

Ruff (0.8.2)

Missing return type annotation for staticmethod `_is_goal_req_in_range`

(ANN205)